Makefile

```diff
@@ -4,13 +4,11 @@ PREFIX ?= /
 
 .PHONY: install
 install:
-	install -D -m0644 src/overdue.awk $(PREFIX)/usr/share/overdue/overdue.awk
 	install -D -m0644 src/overdue-libraries-upgrade.hook \
 		$(PREFIX)/usr/share/libalpm/hooks/overdue-libraries-upgrade.hook
-	install -D -m0755 src/overdue.sh  $(PREFIX)/usr/bin/overdue
```

Comment:
You can pipe a list of library files you want to check into the script. This one pipes all library files located in `/usr/lib`:

`find /usr/lib/*.so -printf '%f\n\0' | /usr/share/libalpm/scripts/overdue`

So perhaps that could work as another script to be put in `PATH`?